dd-sdk-android/src/main/kotlin/com/datadog/android/rum/tracking/MixedViewTrackingStrategy.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.tracking
+
+import android.app.Activity
+import androidx.fragment.app.Fragment
+
+/**
+ * A [ViewTrackingStrategy] that will track [Activity] or [Fragment] as RUM View Events.
+ * This strategy will apply both the [ActivityViewTrackingStrategy]
+ * and the [FragmentViewTrackingStrategy] and will remain for you to decide whether to exclude
+ * some activities or fragments from tracking by providing an implementation for the right
+ * predicate in the constructor arguments.
+ * @see ActivityViewTrackingStrategy
+ * @see FragmentViewTrackingStrategy
+ * **Note**: This version of the [MixedViewTrackingStrategy] is compatible with
+ * the AndroidX Compat Library.
+ * @param trackExtras whether to track Activity Intent extras or the Fragment arguments.
+ * @param componentPredicate to accept the Activities that will be taken into account as
+ * valid RUM View events.
+ * @param supportFragmentComponentPredicate to accept the Androidx Fragments
+ * that will be taken into account as valid RUM View events.
+ * @param defaultFragmentComponentPredicate to accept the default Android Fragments
+ * that will be taken into account as valid RUM View events.
+ */
+class MixedViewTrackingStrategy :
+    ActivityLifecycleTrackingStrategy,
+    ViewTrackingStrategy {
+
+    private val activityViewTrackingStrategy: ActivityViewTrackingStrategy
+    private val fragmentViewTrackingStrategy: FragmentViewTrackingStrategy
+
+    @JvmOverloads
+    constructor(
+        trackExtras: Boolean,
+        componentPredicate: ComponentPredicate<Activity> = AcceptAllActivities(),
+        supportFragmentComponentPredicate: ComponentPredicate<Fragment> =
+            AcceptAllSupportFragments(),
+        defaultFragmentComponentPredicate: ComponentPredicate<android.app.Fragment> =
+            AcceptAllDefaultFragment()
+    ) : this(
+        ActivityViewTrackingStrategy(trackExtras, componentPredicate),
+        FragmentViewTrackingStrategy(
+            trackExtras,
+            supportFragmentComponentPredicate,
+            defaultFragmentComponentPredicate
+        )
+    )
+
+    internal constructor(
+        activityViewTrackingStrategy: ActivityViewTrackingStrategy,
+        fragmentViewTrackingStrategy: FragmentViewTrackingStrategy
+    ) : super() {
+        this.activityViewTrackingStrategy = activityViewTrackingStrategy
+        this.fragmentViewTrackingStrategy = fragmentViewTrackingStrategy
+    }
+
+    // region ActivityLifecycleTrackingStrategy
```

Comment:
Maybe you should override all callbacks ? Otherwise if one implementation decide it needs to act in the onCreate, it'll break when users use the MixedViewTrackingStrategy